envoy-control-core/src/main/resources/lua/ingress_rbac_logging.lua

```diff
@@ -40,6 +51,7 @@ function envoy_on_response(handle)
     local method = lua_metadata["request.info.method"] or ""
     local xff_header = lua_metadata["request.info.xff_header"] or ""
     local source_ip = string.match(xff_header, '[^,]+$') or ""
+    local request_id = lua_metadata["request.info.request_id"] or ""
     local statusCode = handle:headers():get(":status") or "0"
-    handle:logInfo("\nINCOMING_PERMISSIONS { \"method\": \""..method.."\", \"path\": \""..path.."\", \"clientIp\": \""..source_ip.."\", \"clientName\": \""..client_name.."\", \"protocol\": \""..protocol.."\", \"statusCode\": "..statusCode.." }")
+    handle:logInfo("\nINCOMING_PERMISSIONS { \"method\": \""..method.."\", \"path\": \""..path.."\", \"clientIp\": \""..source_ip.."\", \"clientName\": \""..client_name.."\", \"protocol\": \""..protocol.."\", \"requestId\": \""..request_id.."\", \"statusCode\": "..statusCode.." }")
```

Comment:
Sending request with `x-request-id` header with value containing `"` will produce invalid json and the request will not be logged on the log-consumer side. Is this acceptable?

Reply:
This affects all other fields, so you can send a request with path containing `"`. I think we agreed that this is an acceptable trade-off.

Reply:
If we agree that this is acceptable, I will accept it too :)